vtk-cdat/conda_build_config.yaml

```diff
@@ -0,0 +1,17 @@
+python:
+    - 3.6
+    - 2.7
+pin_run_as_build:
+  python: x.x
+
+ffmpeg:
+    - 4.0.*
+pin_run_as_build:
+  ffmpeg: x.x
+
+numpy:
+    - 1.15
+
+VTK_WITH_OSMESA:
```

Comment:
I think you took off the "feature" capability, no? I use to determine this from a "conda list"

Reply:
I think you generated two different meta.yaml from prep_from_build. The advantage of this is that it builds all 4 configurations with one conda build.